tests/integration/ceph_mgr_test.go

```diff
@@ -123,16 +123,32 @@ func (suite *CephMgrSuite) execute(command []string) (error, string) {
 
 func (suite *CephMgrSuite) waitForOrchestrationModule() {
 	var err error
+	var orchStatus map[string]string
 	for timeout := 0; timeout < 30; timeout++ {
 		err, output := suite.execute([]string{"status"})
-		logger.Info("%s", output)
+		logger.Infof("output: %s", output)
 		if err == nil {
 			logger.Info("Rook Toolbox ready to execute commands")
+			// Convert string returned to map
+			output_lines := strings.Split(output, "\n")
+			orchStatus = make(map[string]string)
+			for _, setting := range output_lines {
```

Comment:
To make the test more consistent, I'd suggest we actually remove the declaration of the rook module from the test [here](https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_manifests.go#L2052-L2053). Then the test will always enable the rook module with this code instead of sometimes one way or the other.

Reply:
Thx for the suggestion.!!
After the modification... I have tested several times an it seems to be ok this time. With this change i expect that it is going to be difficult that the test fails because the manager module is not enabled or not set the orch. backend... lets see...

We will have this kind of output starting the test: ( )

```
2020-07-28 15:36:24.022761 I | testutil: waiting for 1 pods (found 0) with label app=rook-ceph-mgr in namespace mgr-ns
2020-07-28 15:36:29.026142 I | testutil: found 1 pods with label app=rook-ceph-mgr
2020-07-28 15:36:29.026155 I | installer: Rook Cluster started
2020-07-28 15:36:29.031819 I | installer: Starting Rook toolbox
2020-07-28 15:36:29.031844 I | testutil: Running kubectl [create -f -]
pod/rook-ceph-tools created
2020-07-28 15:36:34.220329 I | testutil: waiting for pod rook-ceph-tools in namespace mgr-ns to be running
2020-07-28 15:36:34.226700 I | installer: Rook Toolbox started
2020-07-28 15:36:34.226718 I | installer: installed rook operator and cluster : mgr-ns on k8s v1.18.6
2020-07-28 15:36:34.238256 I | integrationTest: Enabling Rook orchestrator module: <ceph mgr module enable rook>
2020-07-28 15:36:34.238278 D | exec: Running command: kubectl exec -i rook-ceph-tools -n mgr-ns -- ceph mgr module enable rook --connect-timeout=15
2020-07-28 15:36:35.802701 I | integrationTest: output: 
2020-07-28 15:36:35.802716 I | integrationTest: Setting orchestrator backend to Rook .... <ceph orch set backend rook>
2020-07-28 15:36:35.802728 D | exec: Running command: kubectl exec -i rook-ceph-tools -n mgr-ns -- ceph orch set backend rook --connect-timeout=15
2020-07-28 15:36:42.479169 I | integrationTest: output: 
2020-07-28 15:36:42.479189 I | integrationTest: Waiting for rook orchestrator module enabled and ready ...
2020-07-28 15:36:42.479208 D | exec: Running command: kubectl exec -i rook-ceph-tools -n mgr-ns -- ceph orch status --connect-timeout=15
2020-07-28 15:36:43.031893 I | integrationTest: output: Backend: rook
Available: True
2020-07-28 15:36:43.031906 I | integrationTest: Rook Toolbox ready to execute commands
=== RUN   TestCephMgrSuite/TestCreateOSD
2020-07-28 15:36:43.032097 I | integrationTest: Testing .... <ceph orch create OSD>
```